ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -57,6 +57,120 @@ presubmits:
           name: prow-job-cluster-launch-installer-e2e
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws-op
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    name: pull-ci-openshift-machine-config-operator-master-e2e-aws-op
+    rerun_command: /test e2e-aws-op
```

Comment:
Might want to also add the regex from #2576 here too.

Reply:
AFAICS now that part isn't in the generated job (I think it's a configmap)?

Reply:
It's higher up in that same file, no? It looks like it's working too. E.g. the CI status in https://github.com/openshift/machine-config-operator/pull/306 for that single commit says:

> ci/prow/e2e-aws — Skipped

Reply:
Oh sorry I misunderstood; you want to also skip this test for docs (which makes sense).  I thought you were saying there'd be a merge conflict.